### DISCUSSION TITLE
Why don't my docker ui commands work?How do I set the UI port number on the docker-compose.yaml file?

### DISCUSSION THREAD

**Discussion by Tk8and15** [geldata/gel#4730] [Q&A]
![image](https://user-images.githubusercontent.com/118907993/204417195-3cc2d3c1-2e75-412a-a096-22168fa14e86.png)
![image](https://user-images.githubusercontent.com/118907993/204417225-9a16e0cc-7511-4b46-935f-6d685bf8eb42.png)
![image](https://user-images.githubusercontent.com/118907993/204417238-65630641-8309-436a-9269-f7908088af17.png)

**Comment by Sikarii** [comment]
Hey, you can enable the UI with an environment variable (in addition to the command line):
`EDGEDB_SERVER_ADMIN_UI: enabled` in your docker-compose environment should do.

**Comment by Tk8and15** [comment]
Thank you, my friend!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "edgedb/edgedb"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月29日(星期二) 上午10:00
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [edgedb/edgedb] Why don't my docker ui commands work?How do I set the UI port number on the docker-compose.yaml file? (Discussion #4730)





 
Hey, you can enable the UI with an environment variable (in addition to the command line):
 EDGEDB_SERVER_ADMIN_UI: enabled in your docker-compose environment should do.
